politeiawww/api/comments/v1/v1.go

```diff
@@ -338,12 +338,6 @@ type DelReply struct {
 	Comment Comment `json:"comment"`
 }
 
-const (
```

Comment:
We can't remove these right now since it's public API and is a breaking change.  At some point we'll be deleting all of these politeiawww plugin APIs as part of a major version upgrade.  Until then, you'll need to keep these in and make a comment explaining that they're depracted and where the correct page sizes can be found.